lib/pharos/resources/weave/daemon-set.yml.erb

```diff
@@ -38,9 +38,6 @@ spec:
                 secretKeyRef:
                   name: weave-passwd
                   key: weave-passwd
-            # defaults to 0.0.0.0:6782, limit to block public access
-            - name: WEAVE_STATUS_ADDR
-              value: 127.0.0.1:6782
```

Comment:
Alternatively we could switch to `WEAVE_METRICS_ADDR=127.0.0.1:6782` to lock this down even further... would need to know how we intend to scrape Prometheus metrics from the nodes.